test_runner/src/main/kotlin/ftl/android/AndroidCatalog.kt

```diff
@@ -31,7 +32,11 @@ object AndroidCatalog {
 
     fun supportedOrientationsAsTable(projectId: String) = deviceCatalog(projectId).runtimeConfiguration.orientations.asPrintableTable()
 
-    fun localesAsTable(projectId: String) = deviceCatalog(projectId).runtimeConfiguration.locales.asPrintableTable()
+    fun localesAsTable(projectId: String) = getLocales(projectId).asPrintableTable()
+
+    fun getLocaleDescription(projectId: String, locale: String) = getLocales(projectId).getLocaleDescription(locale)
+
+    private fun getLocales(projectId: String) = deviceCatalog(projectId).runtimeConfiguration.locales
```

Comment:
I am wondering why do we need `projectId` to make this request? I mean, I can see from the code that we use it to set a property for request under the hood, but why since it is publicly available information, maybe it's due to 'default' tag...I don't know and just thinking :)

Reply:
Probably device catalogue could be different for different project ids, so API needs this to return catalogue which is available for current `projectID`.
I played some time with documentation and API tester and I think that is the best explanation

Reply:
@Pasterzp requested change done. Thanks for the tests. I check if projectID is really required and get feedback

Reply:
As per [documenation](https://cloud.google.com/sdk/gcloud/reference#--project):

`--project` and its fallback` core/project` property play two roles in the invocation. It specifies the project of the resource to operate on. 

It has fallback option, so project is not specified in environmental variable and not provided by config for me it is ok to use empty

Reply:
hey, we need projectId

the reason is select projects have access to secret device catalogs 🙂 These catalogs are not available to the public.

Reply:
> Probably device catalogue could be different for different project ids,

this is correct!

Reply:
Let's add a comment that explains why projectId is sent to the device catalog